### DISCUSSION TITLE
Sales Target issue

### DISCUSSION THREAD

**Discussion by samuelsituma** [YetiForceCompany/YetiForceCRM#14255] [How to]
How do I add record sales targets for users in different periods and record actual sales then compare if the user met the target we can reward them.

**Comment by deepti4666** [comment]
Would love to know as well, is this a custom function?

**Comment by davide-alghi** [comment]
Hi @samuelsituma,
you could create a custom module "User Targets" with fields like: employee, start date, end date, qty, product/service

**Comment by samuelsituma** [comment]
Thank you let me try creating a custom module and see how it goes
